### PR TITLE
[#50658] Connect to OpenProject Button in Smart Picker

### PR DESCRIPTION
**Pull request by SwikritiT**
https://community.openproject.org/work_packages/50658

Screenshot from 2023-12-04 14-29-17

> Note: that the redirection does take back to the talk chat but it doesn't open the smart picker itself, as there was no easy of opening the smart picker programmatically, we decided that it's okay that the redirection doesn't open the smart picker. The user would have to open the smart picker to know if the OAuth connection was successful. Similar, is the reason for the lack of success/failure message, as we'll have to mount the smart picker component to display success/failure message for the connection, there was no easy way to mount it during the OAuth redirection so we decided that we'll leave it as it is in this PR

### CODE REVIEW

File: src/views/WorkPackagePickerElement.vue [hunk 1]
```diff
@@ -24,16 +24,22 @@
...
+			:is-disabled="isLoading || !isAdminConfigOk || !isStateOk"
 			:linked-work-packages="linkedWorkPackages"
 			@submit="onSubmit" />
-		<EmptyContent id="openproject-empty-content"
-			:state="state"
-			:file-info="fileInfo"
-			:is-smart-picker="true"
-			:is-admin-config-ok="isAdminConfigOk" />
+		<div id="openproject-empty-content">
+			<NcLoadingIcon v-if="isLoading" class="loading-spinner" :size="50" />
```
[the remark this answers is not in the document]
Comment (SwikritiT) [reply src/views/WorkPackagePickerElement.vue:35]: I changed the size to 100 can you check again ?

[the remark this answers is not in the document]
Comment (SagarGi) [reply src/views/WorkPackagePickerElement.vue:35]: size 90 LGTM after i checked.

**Review by SagarGi — DISMISSED** [review]
LGTM :+1:

**Review by SagarGi — CHANGES_REQUESTED** [review]
unexpectedError
I got this after cutting openproject after successful OAUTH connection.

**Comment by SwikritiT** [comment]
> unexpectedError I got this cutting openproject after successful OAUTH connection.

fixed it can you check again?

**Review by SagarGi — APPROVED** [review]
LGTM :+1:

**Pull request merged**